the-events-calendar.php

```diff
@@ -29,7 +29,7 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-define( 'TRIBE_EVENTS_FILE', __FILE__ );
+define( 'TRIBE_EVENTS_FILE' , __FILE__ );
```

Comment:
:no_entry_sign: Space found before comma in function call

<sub>hash: cef9386b870da5045c5568763ca9a685</sub>